### PR TITLE
FE-1741 - table styling bug (action toolbar)

### PR DESCRIPTION
**Pull request by izakrzewska**
# Description
Action Toolbar rewritten to styled-components.
Component updated to display correctly when in use with the DLS table.

# Screenshots / Gifs
Classic theme
Zrzut ekranu 2019-07-3 o 14 34 59
Zrzut ekranu 2019-07-3 o 14 35 41

DLS
Zrzut ekranu 2019-07-3 o 14 35 53
Zrzut ekranu 2019-07-3 o 14 36 05

### CODE REVIEW

File: src/components/action-toolbar/action-toolbar.component.js [hunk 1]
```diff
@@ -34,36 +34,19 @@ import './action-toolbar.scss';
...
  *  Additional props for Link or Icon can be passed in the action object.
  *
- * @class ActionToolbar
- * @constructor
  */
 class ActionToolbar extends React.Component {
   // TODO This component needs to be freestanding - we need to provide an api that allows it be used independently.
   // https://github.com/Sage/carbon/issues/1070
 
   static propTypes = {
```
Comment (dawidzarzycki) [inline src/components/action-toolbar/action-toolbar.component.js]: Would you like to move the propTypes out of the static?
`ActionToolbar.propTypes = {}`

File: src/components/table/table-story-helpers/table-story-wrapper.component.js [hunk 2]
```diff
@@ -166,11 +166,11 @@ const Wrapper = (props) => {
           path='/countries'
           actions={ {
             delete: {
-              icon: 'bin',
```
Comment (edleeks87) [inline src/components/table/table-story-helpers/table-story-wrapper.component.js]: @harpalsingh/ @benwilsonsage  are these the correct icons?

**Comment by harpalsingh** [comment]
Could we change the buttons in the action toolbar to be the small size.

Currently:

Screenshot 2019-07-08 at 09 57 44

Change to the small button size please.

**Comment by harpalsingh** [comment]
Passing UX review for now, though the button sizes should be small not medium, but since multiaction button has not got the scale for sizes avaliable, we need to keep medium for now and this will need to be a future ticket to correct.

**Pull request merged**